### PR TITLE
[POAE7-1220] support Spark+Plasma DS cache on K8S

### PR DESCRIPTION
**Pull request by yma11**
Use Plasma DS cache with Spark on K8S

### CODE REVIEW

**Comment by yma11** [comment]
@zhixingheyi-tian @winningsix Please help take a review.

File: integrations/oap/kubernetes/oap/sql-ds-cache/README.md [hunk 1]
```diff
@@ -0,0 +1,152 @@
+# Use Plasma SQL DS CACHE 
+### 1. Start Plasma Store Server POD
+```
+// plasma-store-server.yaml 
+apiVersion: apps/v1
```
Comment (winningsix) [inline integrations/oap/kubernetes/oap/sql-ds-cache/README.md:5]: small hints: maybe skip some content since duplicated in yml

**Comment by winningsix** [comment]
LGTM

File: integrations/oap/kubernetes/oap/sql-ds-cache/plasma-store-server.yaml [hunk 2]
```diff
@@ -0,0 +1,42 @@
...
+      name: pmem
+  template:
+    metadata:
+      labels:
+        name: pmem
+    spec:
+      containers:
+      - name: plasma-store-server
+        image: oap-centos:1.2.0
+        imagePullPolicy: IfNotPresent
```
Comment (zhixingheyi-tian) [inline integrations/oap/kubernetes/oap/sql-ds-cache/plasma-store-server.yaml:20]: make the image passed by parameters

File: integrations/oap/kubernetes/oap/sql-ds-cache/plasma-store-server.yaml [hunk 3]
```diff
@@ -0,0 +1,42 @@
...
+        command: ["/bin/sh"]
+        args: ["-c", "export LD_LIBRARY_PATH=/opt/home/conda/envs/oap-1.2.0/lib:$LD_LIBRARY_PATH; echo $LD_LIBRARY_PATH; export PMEM_PATH=/mnt/pmem; export PMEM_SIZE=5000000000; /opt/home/conda/envs/oap-1.2.0/bin/plasma-store-server -m $PMEM_SIZE -d $PMEM_PATH -s /var/log/plasmaStore;"]
+        resources:
```
Comment (zhixingheyi-tian) [inline integrations/oap/kubernetes/oap/sql-ds-cache/plasma-store-server.yaml:23]: Also can export /opt/home/conda/envs/oap-1.2.0 to env;
Make PMEM_SIZE passed by command

**Pull request merged**